os/arch/arm/src/stm32/stm32_otgfshost.c

```diff
@@ -768,23 +802,23 @@ static void stm32_chan_configure(FAR struct stm32_usbhost_s *priv, int chidx)
 
 	/* Program the HCCHAR register */
 
-	regval = ((uint32_t)priv->chan[chidx].maxpacket << OTGFS_HCCHAR_MPSIZ_SHIFT) | ((uint32_t)priv->chan[chidx].epno << OTGFS_HCCHAR_EPNUM_SHIFT) | ((uint32_t)priv->chan[chidx].eptype << OTGFS_HCCHAR_EPTYP_SHIFT) | ((uint32_t)priv->devaddr << OTGFS_HCCHAR_DAD_SHIFT);
+	regval = ((uint32_t) chan->maxpacket << OTGFS_HCCHAR_MPSIZ_SHIFT) | ((uint32_t) chan->epno << OTGFS_HCCHAR_EPNUM_SHIFT) | ((uint32_t) chan->eptype << OTGFS_HCCHAR_EPTYP_SHIFT) | ((uint32_t) chan->funcaddr << OTGFS_HCCHAR_DAD_SHIFT);
```

Comment:
Remove the space between typecasting and variable.

Reply:
Done